app/models/manageiq/providers/microsoft/inventory/collector/infra_manager.rb

```diff
@@ -59,6 +59,23 @@ def hosts
     hosts_by_host_name.values
   end
 
+  def storage_id_by_host_name_and_mount_point
+    @storage_id_by_host_name_and_mount_point ||= begin
+      drive_letter = /\A[a-z][:]/i
+
```

Comment:
I can't remember, is a UNC path a possibility here?

Reply:
Possibly?  I lifted this from the existing logic.  For now I'm just looking for exact equality between the refresh strategies.

Reply:
Ok, if it hasn't come up before then don't worry about it for now. :)